docs/docs/developers/tutorials/codealong/contract_tutorials/token_contract.md

```diff
@@ -314,7 +314,7 @@ After initializing storage, the function checks that the `msg_sender` is authori
 
 TODO: update from `prepare_transfer_to_private`
```

Comment:
This method actually does not exist on the token contract anymore. Should we remove this page since it's no longer published? If we need it we can get it from git history